contracts/UsingWitnet.sol

```diff
@@ -32,54 +29,61 @@ abstract contract UsingWitnet {
     _;
   }
 
- /**
-  * @notice Send a new request to the Witnet network with transaction value as result report reward.
-  * @dev Call to `post_dr` function in the WitnetRequestBoard contract.
-  * @param _request An instance of the `Request` contract.
-  * @return Sequencial identifier for the request included in the WitnetRequestBoard.
-  */
-  function witnetPostRequest(Request _request) internal returns (uint256) {
-    return wrb.postDataRequest{value: msg.value}(address(_request));
-  }
-
- /**
-  * @notice Check if a request has been resolved by Witnet.
-  * @dev Contracts depending on Witnet should not start their main business logic (e.g. receiving value from third.
-  * parties) before this method returns `true`.
-  * @param _id The unique identifier of a request that has been previously sent to the WitnetRequestBoard.
-  * @return A boolean telling if the request has been already resolved or not.
-  */
+  /**
+   * @notice Check if a request has been resolved by Witnet.
+   * @dev Contracts depending on Witnet should not start their main business logic (e.g. receiving value from third.
+   * parties) before this method returns `true`.
+   * @param _id The unique identifier of a request that has been previously sent to the WitnetRequestBoard.
+   * @return A boolean telling if the request has been already resolved or not.
+  **/
```

Comment:
What's the result of `witnetCheckRequestResolved` if the request has been resolved and then destroyed?

Reply:
`false`.

Considering that only the `UsingWitnet` that posted the Witnet data request can actually destroy it, the `witnetCheckRequestResolved` should be interpreted as "check the given request hast already been solved by Witnet, but not yet destroyed by its requestor".

Reply:
I would add a comment saying that after destroying the result `witnetCheckRequestResolved` will return false.